serial_test_derive/src/lib.rs

```diff
@@ -169,3 +190,41 @@ fn test_stripped_attributes() {
     };
     assert_eq!(format!("{}", compare), format!("{}", stream));
 }
+
+#[test]
+fn test_serial_async() {
+    let attrs = proc_macro2::TokenStream::new();
+    let input = quote! {
+        #[test]
+        async fn foo() {}
+    };
+    let stream = serial_core(attrs.into(), input);
+    let compare = quote! {
+        #[test]
+        async fn foo () {
+            serial_test::async_serial_core("", || async {
+                {}
+            }).await;
+        }
+    };
+    assert_eq!(format!("{}", compare), format!("{}", stream));
+}
+
+#[test]
+fn test_serial_async_return() {
+    let attrs = proc_macro2::TokenStream::new();
+    let input = quote! {
+        #[test]
```

Comment:
```suggestion
        #[actix_rt::test]
```

Reply:
ok, I thought that by this we can show that we do care for other stuff than just plain tokio. :) And mybe for better searchability in serach engines :)